change-logs in 2026 for edge (#4505)
SoftwareAG/c8y-docs: content/change-logs/edge/edge-2026.0.0-removal-external-mongo.md

---
date: 2026-01-01
title: Edge no longer supports externally hosted MongoDB
change_type:
  - value: change-inv-3bw8e
    label: Announcement
product_area: Edge
component:
  - value: component-g7hnfbu4J
    label: Edge
build_artifact:
  - value: tc-nJH2U7g3u
    label: edge
ticket: CIE-5542
version: 2026.0.0
---

> To improve reliability and reduce operational complexity, support for external MongoDB deployments has been removed. We recommend using the default bundled MongoDB setup for all {{< product-c8y-iot >}} Edge deployments.

Reviewer: 'Recommend' is too soft a word when there's no choice :-D
```suggestion
To improve reliability and reduce operational complexity, support for external MongoDB deployments has been removed. All {{< product-c8y-iot >}} Edge deployments will use the default bundled MongoDB setup.
```
